### PR TITLE
When running a server, don't load minimap previews into memory.

### PR DESCRIPTION
**Pull request by RoosterDragon**
This reduces memory usage when starting up a server for the ra mod from 113 MB to 88 MB.

### CODE REVIEW

File: OpenRA.Server/Program.cs [hunk 1]
```diff
@@ -83,6 +83,7 @@ static void Run(string[] args)
 			{
 				// HACK: The engine code *still* assumes that Game.ModData is set
 				var modData = Game.ModData = new ModData(mods[modID], mods);
+				modData.MapCache.LoadPreviewImages = false;
```
Comment (PunkPun) [inline OpenRA.Server/Program.cs]: It might be worth adding a PERF comment.

**Comment by pchote** [comment]
Would this be helpful for the utility too?

**Comment by PunkPun** [comment]
some commands may require maps map previews, that would need more investigation

**Comment by ThomasChr** [comment]
LGTM

File: OpenRA.Server/Program.cs (hunk 1 above)
Comment (RoosterDragon) [reply OpenRA.Server/Program.cs]: Added

**Review by PunkPun — APPROVED** [review]

**Comment by RoosterDragon** [comment]
> Would this be helpful for the utility too?

I can't locate any call sites from the utilities that would end up loading preview images, so it would seem not.

**Review by ThomasChr — APPROVED** [review]

**Comment by penev92** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/fc2d0583425cb1a00ceefe8849760c5386a92f68...acd291d73a352118362d31aea1d3677221cf89dd)

[Commit on `prep-2211`](https://github.com/OpenRA/OpenRA/commit/12965fc1257327b65cbab84eb78a6dc53316feee)

**Pull request merged**